src/library/base/man/is.finite.Rd

```diff
@@ -85,11 +86,12 @@ NaN
   \code{dimnames} and \code{names} attributes are preserved.
 }
 \seealso{
-  \code{\link{NA}}, \sQuote{\emph{Not Available}} which is not a number
-  as well, however usually used for missing values and applies to many
-  modes, not just numeric and complex.
+  \code{\link{NA}}, \sQuote{\emph{Not Available}} / \sQuote{Missing value} 
+  which is a a logical constant of length 1 which contains a missing
+  value indicator. \code{\link{NA}} can be coerced to numeric as
+  \code{\link{NA_integer_}} and \code{\link{NA_real_}}.
```

Comment:
Not sure about adding \sQuote{Missing value} (that is referred to in the next phrase already.

 \code{\link{NA}}, \sQuote{\emph{Not Available}}, which is usually used to denote a missing value. The default mode of \code{NA} is \code{logical}; unlike \code{NaN}, which is always of mode \code{double}, there are equivalent missing values  with different storage modes (\code{NA_real_}, \code{NA_integer_},\code{NA_complex_}, and \code{NA_character_}, all of which are displayed as \code{NA}).
     usually used for missing values and applies to many modes, not
     just numeric and complex.

Reply:
It's hard to figure out how to include enough detail here so that people don't get confused, without repeating too much from `?NA` ...

Reply:
I used the beginning from your suggestion to make it shorter